docker/Dockerfile.base

```diff
@@ -40,7 +40,7 @@ RUN mkdir -m0755 /licenses
 COPY ./LICENSE-Apache /licenses/apache.txt
 
 ### Add necessary Red Hat repos and packages
-RUN INSTALL_PKGS="iproute procps" && \
+RUN INSTALL_PKGS="iproute procps shadow-utils" && \
     microdnf -y update --setopt=install_weak_deps=0 --setopt=tsflags=nodocs && \
     microdnf -y install --setopt=install_weak_deps=0 --setopt=tsflags=nodocs ${INSTALL_PKGS}
```

Comment:
I guess `shadow-utils` should only be removed if it isn't in `${INSTALL_PKGS}`. Alternatively we could install in in the `fetch-ziti-artifacts` stage and `COPY --from=fetch-ziti-artifacts` the modified group files (/etc/group, /etc/gshadow) from the stage.